cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -252,10 +251,6 @@ friendlyTxBodyImpl
                     era
                     (`getRedeemerDetails` tb)
                  )
-              ++ ( monoidForEraInEon @AlonzoEraOnwards
-                    era
-                    (`friendlySupplementalDatums` txSupplementalData)
-                 )
```

Comment:
Why are we dropping this?

Reply:
I'm wondering if the removal of the field has changed the tx size, and the fee. Has the fee changed in cardano-testnet tests?

Reply:
@carbolymer> I didn't test this change with `cardano-testnet` yet. I will do that soon :+1: